src/mixer/basetrackplayer.h

```diff
@@ -41,6 +42,8 @@ class BaseTrackPlayer : public BasePlayer {
     void loadTrackFailed(TrackPointer pTrack);
     void newTrackLoaded(TrackPointer pLoadedTrack);
     void unloadingTrack(TrackPointer pAboutToBeUnloaded);
+    void noPassthroughInputConfigured();
+    void noVinylControlInputConfigured();
 };
 
 class BaseTrackPlayerImpl : public BaseTrackPlayer {
```

Comment:
At the time I wrote the test suite I didn't want to change every instance of BaseTrackPlayer in the codebase to BaseTrackPlayerAbstract/Interface. I don't see a strong need to do any name changing here.